src/Uno.UI.Composition/Composition/ShapeVisual.skia.cs

```diff
@@ -54,4 +55,27 @@ internal override void Paint(in PaintingSession session)
 
 		return shape;
 	}
+
+	/// <remarks>This does NOT take the clipping into account.</remarks>
```

Comment:
Is it applied somewhere else? If not this would introduce regression on skia I think

Reply:
Yes. the way hit-testing works in this PR is that a visual hit-tests positively checks the clip area as before AND also HitTest returns true. This is why we don't need to take clipping into account here.

Reply:
Great, this match what we did here https://github.com/unoplatform/uno/pull/17633/commits/f5c4c7c43d86413f3596c1322c42353b8464e837# 👍 (Once both PR have been merged, we will have to converge APIs)